src/main/kotlin/com/jaredsburrows/license/internal/report/HtmlReport.kt

```diff
@@ -32,6 +38,10 @@ class HtmlReport(private val projects: List<Project>) {
     const val NO_LIBRARIES = "None"
     const val NO_LICENSE = "No license found"
     const val NOTICE_LIBRARIES = "Notice for packages:"
+    const val COPYRIGHT = "Copyright "
+    const val DEFAULT_AUTHOR = "The original author or authors"
+    const val DEFAULT_YEAR = "20xx"
```

Comment:
Will this show up if no year is specified?

Reply:
Lines 42 and 43 are default text in case the information is absent in the POM. My thought was to leave enough structure there that it's easy to hand-edit any corrections if the user wants to go to that effort. Removal of the whole thing is easily done if that's the preference.

I'll push an update shortly.